bench/workgen/runner/evict-btree-lookaside.py

```diff
@@ -119,7 +119,7 @@
         compress_table_config + table_config + ",log=(enabled=true)")
 log_table = Table(log_name)
 
-ops = Operation(Operation.OP_SEARCH, tables[0])
+ops = Operation(Operation.OP_SEARCH, tables[0],Key(Key.KEYGEN_PARETO, 0, ParetoOptions(1)))
```

Comment:
This will increase the probability that search thread will hit a key that is part of long running transaction.